### ISSUE TITLE
Integrate with Wayback machine

### ISSUE THREAD

**Issue by simonsan** [cadeyrn/bookmarks-organizer#147]
I think it could be useful to check back with Wayback machine (web.archive.org) before deleting a bookmark completely. It might be even worth to rewrite the bookmark then, if the page is existing there.

**Comment by dave669** [comment]
yes, please. I do this manually with any links that I think are "retired". I also add "(Archive)" at the end of the bookmark text to signify it is not a current link. It would be great if this feature could also optionally append a text string to bookmarks re-written to web.archive.org links.

**Comment by dave669** [comment]
Hi,
     No, I don't know of any others with this feature. I know of some 
that have the feature of re-writing the bookmark name so that it is in a 
google search "box" allowing one to click on it and maybe find a new 
location for that bookmark if it kept the same name and is in google. 
This feature would be similar, it could just rewrite the URL of the 
bookmark to have the www.archive.com url in front of it, so the user 
could click on it to see if wayback machine will give results. Or even 
do this search automatically for the user. Also, the ability to change 
the bookmark name, i.e. add "(Archive)" at the end, would be great, too.
     For example:
Getting Started with OpenVMS
http://www.alphajockey.com/getting_started.html

     To:
Getting Started with OpenVMS (Archive)
https://web.archive.org/web/2005*/http://www.alphajockey.com/getting_started.html


Thanks,
Dave

**Comment by simonsan** [comment]
URLs can be tested for a `200` by replacing the initial URL with `https://web.archive.org/web/<URL>`. The date is not needed, the aforementioned URL directs to the latest snapshot stored. I agree that an addition of `(Archive)` would be nice.

**Comment by obrientg** [comment]
Yes! I was coming to request this same functionality: Being able to find that web site or that specific web page on archive.org that was originally bookmarked would be a tremendous help. 

+1 for this functionality please - TUVM!
